src/screens/HomeScreen/HowYouFeel.js

```diff
@@ -0,0 +1,119 @@
+import React, { Component } from 'react';
+import {
+  Text,
+  View,
+  AsyncStorage,
+  StyleSheet,
+  FlatList,
+  Image,
+  Dimensions,
+  TouchableOpacity,
+} from 'react-native';
+
+export default class HowYouFeel extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      emoticons: [
+        require('../../assets/images/Emoji/neutral.png'),
+        require('../../assets/images/Emoji/happy.png'),
+        require('../../assets/images/Emoji/cry.png'),
+        require('../../assets/images/Emoji/fearful.png'),
+        require('../../assets/images/Emoji/love_hearts.png'),
+        require('../../assets/images/Emoji/tired_face.png'),
+        require('../../assets/images/Emoji/unamused.png'),
+        require('../../assets/images/Emoji/disappointed.png'),
+      ],
+    };
+  }
+
+  saveData = async item => {
+    var emoList = [
+      'neutral',
+      'happy',
+      'cry',
+      'fearful',
+      'love_hearts',
+      'tired_face',
+      'unamused',
+      'disappointed',
+    ];
+    var emotion = emoList[item];
```

Comment:
Don't use `var`
use `let` or `const`
use `let` if value can change.
use `const` if the value is not changed during the execution.